src/main/java/com/jcabi/github/RtReleases.java

```diff
@@ -113,6 +112,30 @@ public Release get(final int number) {
     }
 
     @Override
+    public Release tagged(final String name) throws IOException {
+        return this.get(
+                this.request.uri().path("/tags/").path(name)
+                        .back().method(Request.GET)
+                        .fetch().as(RestResponse.class)
+                        .assertStatus(HttpURLConnection.HTTP_CREATED)
+                        .as(JsonResponse.class)
+                        .json().readObject().getInt("id")
+        );
+    }
+
+    @Override
+    public Release latest() throws IOException {
+        return this.get(
+                this.request.uri().path("/latest")
```

Comment:
@weisebrazil 4 spaces indentation only here, following lines should also have smaller indentation